src/components/editgrid/EditGrid.js

```diff
@@ -501,10 +502,27 @@ export default class EditGridComponent extends NestedArrayComponent {
         }
         else {
           this.alert.showErrors(editRow.errors, false);
+          editRow.alerts= true;
         }
       },
     }, this.component.saveRow || 'Save'));
 
+    this.on('componentChange', () => {
```

Comment:
This may cause some serious performance issues.  I would rather do this within the ```checkData``` loop which is called every change cycle.